src/database/accessors/loadObjects.ts

```diff
@@ -1,26 +1,14 @@
 import { db } from '../db';
-import { InternalValidationError } from '../../errors';
 import type { TinyPgParams } from 'tinypg';
-import type { ValidateFunction } from 'ajv';
 
 export const loadObjects = async <T extends object>(
   tinyPgQueryName: string,
   tinyPgQueryParameters: TinyPgParams,
-  objectValidator: ValidateFunction<T>,
 ): Promise<T[]> => {
   const { rows } = await db.sql<T>(
     tinyPgQueryName,
     tinyPgQueryParameters,
   );
 
-  rows.forEach((row) => {
```

Comment:
We don't want this check anyway (though we should make sure we're OK with the lack of schema-based coersion, as the validators were performing coersion / ensuring that the resulting properties types aligned with the expected type.)

Reply:
This is a great callout! I don't have all the relevant pieces in my head - does this mean that any `{ "type": "number" }` fields will start returning `string`s instead? Could/should we still apply these type conversions via validation on the narrowed objects?